src/patternfly/components/Accordion/accordion.scss

```
@@ -85,6 +87,11 @@
 
   background-color: var(--#{$accordion}--BackgroundColor);
 
+  &.pf-m-start {
+    --#{$accordion}__toggle--JustifyContent: start;
+    --#{$accordion}__toggle--ColumnGap: var(--#{$pf-global}--spacer--md);
```

I'd create new local component vars for these values and use those vars here. That will allow someone to change the column-gap, for example, by doing this

```
.pf-v5-c-accordion {
  --#{$accordion}--m-toggle-start__toggle--ColumnGap: 42px; // or whatever their custom column gap is
}
```

As opposed to what's in this PR, they'd need to do

```
.pf-v5-c-accordion.pf-m-start {
  --#{$accordion}--m-toggle-start__toggle--ColumnGap: 42px; // or whatever their custom column gap is
}
```

This keeps all the themable vars scoped to the base component class, so they can make all of their var overrides in `.pf-v5-c-accordion` and not having to mess with additional selectors.

```suggestion
    --#{$accordion}__toggle--JustifyContent: var(--#{$accordion}--m-toggle-start__toggle--JustifyContent);
    --#{$accordion}__toggle--ColumnGap: var(--#{$accordion}--m-toggle-start__toggle--ColumnGap);
```